README.md

```diff
@@ -364,7 +364,7 @@ Hurray! You successfully made a contribution!
   - [Rohit Vuppuluri](https://github.com/rohitv-dev)
   - [Russ Perry](https://github.com/rperry99)
   - [Rutik Bhoyar](https://github.com/Rutikab12)
-
+  - [Rakesh](https://github.com/RakeshPotnuru)
```

Comment:
Please add your name in alphabetical order. Your name should be around the top of the `R` section